inc/ThirdParty/Plugins/Optimization/AMP.php

```diff
@@ -115,4 +117,20 @@ public function disable_options_on_amp() {
 			remove_filter( 'wp_calculate_image_srcset', 'rocket_protocol_rewrite_srcset', PHP_INT_MAX );
 		}
 	}
+
+	/**
+	 * Rewrites URLs to the CDN URLs if allowed.
+	 * Rewrites URLs in srcset attributes if allowed.
+	 *
+	 * @since 3.5.5
+	 *
+	 * @param  string $html HTML content.
+	 * @return string       HTML content.
+	 */
+	public static function rewrite_cdn( $html ) {
```

Comment:
Why `static`?

Reply:
I used first the Static class method and then I converted it to instance method but I forgot to remove the static.